src/config/ci.ts

```diff
@@ -0,0 +1,30 @@
+import { ConfigInterface, } from './config';
+
+const config: ConfigInterface = {
+  http: {
+    port: 8088,
+  },
+  modules: {
+    latestVersion: '0.0.7',
```

Comment:
Hmm probably this should be already 0.0.8?

Reply:
ah yes